caibot/ChannelPointRedemptions.js

```diff
@@ -0,0 +1,60 @@
+const FileHandler = require('./io-handler');
+const RegexHelper = require('./regexs');
+
+let rewardsJSON = FileHandler.getRewards();
+let knownCPRewards = new Map(Object.entries(rewardsJSON));
```

Comment:
This doesn't work as intended. It does not save to the map when starting the program.